src/rtf-plugins/fixture-managers/yarpplugin/YarpPluginFixture.h

```diff
@@ -29,10 +29,11 @@ class YarpPluginFixture : public RTF::FixtureManager {
     virtual bool setup(int argc, char** argv);
     virtual bool check();
     virtual void tearDown();
-    bool scanPlugins(yarp::os::ConstString name);
+    bool scanPlugin(yarp::os::ConstString name, yarp::os::ConstString type="");
```

Comment:
Nothing, I simply did a mess with plural and singular, it's better if I put all plural, this function included :sweat_smile: